public/ng-views/maps.html

```diff
@@ -394,36 +395,100 @@ <h6 ng-repeat="(key, variant) in variants" ng-show="variant.sel">
 </leaflet>
 <leaflet  center   = "center_right"
           defaults = "defaults_right"
-          maxbounds = "maxBounds_right"
           id       = "map_right"
           tiles    = "tiles_right"
           >
 </leaflet>
 
 
-<div class="basecol right" ng-show="overview" style="max-height: 80%">
+<!--CONTROL PANEL ROUTE SELECTOR MODULE--> 
+
+<div class="basecol_routeSelector col-md-2" >
```

Comment:
This is bad - we do not want to have specific overwrites on `col-md-2` because that is a totally different structure from Bootstrap. You are essentially using none of the inherited components so there is no need for you to be using this class. Furthermore, this class is going to have side effects you are unaware of so including it is in fact dangerous since you want to explicitly accommodate those yourself.